### PR TITLE
Fix issue DPTP-3312 Add oc and hypershift CLI

### PR DESCRIPTION
**Pull request by bear-redhat**

### CODE REVIEW

**Comment by bear-redhat** [comment]
/override ci/prow/snyk-deps
Snyk out of quota

File: images/dptp-3312-hypershift-leaks-cleaner/Dockerfile [hunk 1]
```diff
@@ -1,7 +1,14 @@
+FROM registry.ci.openshift.org/ci/hypershift-cli:latest AS hypershift
+FROM registry.ci.openshift.org/ocp/4.14:cli AS oc
```
Comment (hongkailiu) [inline images/dptp-3312-hypershift-leaks-cleaner/Dockerfile:2]: Those multi-stage builds can be achieved in ci-op's config. You will see examples there.

File: images/dptp-3312-hypershift-leaks-cleaner/Dockerfile [hunk 2]
```diff
@@ -1,7 +1,14 @@
...
+
 FROM quay.io/centos/centos:stream8
 LABEL maintainer="bechen@redhat.com"
 
 RUN yum install -y unzip jq
+
+COPY --from=hypershift /usr/bin/hypershift /usr/bin/hypershift
```
Comment (hongkailiu) [inline images/dptp-3312-hypershift-leaks-cleaner/Dockerfile:9]: `--from` could be removed in the suggested way above.

Comment (hongkailiu) [inline images/dptp-3312-hypershift-leaks-cleaner/Dockerfile:9]: With https://github.com/openshift/release/pull/42025

```suggestion
COPY hypershift /usr/bin/hypershift
```

File: images/dptp-3312-hypershift-leaks-cleaner/Dockerfile [hunk 3]
```diff
@@ -1,7 +1,14 @@
...
+COPY --from=oc /usr/bin/oc /usr/bin/oc
```
Comment (hongkailiu) [inline images/dptp-3312-hypershift-leaks-cleaner/Dockerfile:10]: Same here.

**Review by hongkailiu — APPROVED** [review]

File: images/dptp-3312-hypershift-leaks-cleaner/Dockerfile [hunk 4]
```diff
@@ -2,6 +2,10 @@ FROM quay.io/centos/centos:stream8
 LABEL maintainer="bechen@redhat.com"
 
 RUN yum install -y unzip jq
+
+COPY hypershift /usr/bin/hypershift
+COPY cli /usr/bin/oc
```
Comment (hongkailiu) [inline images/dptp-3312-hypershift-leaks-cleaner/Dockerfile:7]: /hold

**Pull request merged**